src/element/bounds.ts

```diff
@@ -342,3 +342,38 @@ export const getResizedElementAbsoluteCoords = (
     maxY + element.y,
   ];
 };
+
+export const getElementPointsCoords = (
+  element: ExcalidrawLinearElement,
+  points: readonly (readonly [number, number])[],
+): [number, number, number, number] => {
+  /*
+  // This might be computationally heavey
+  const gen = rough.generator();
+  const curve = gen.curve(
+    points as [number, number][],
+    generateRoughOptions(element),
+  );
+  const ops = getCurvePathOps(curve);
+  const [minX, minY, maxX, maxY] = getMinMaxXYFromCurvePathOps(ops);
+  */
+  // This doesn't take into account curves with roughjs
+  const { minX, minY, maxX, maxY } = points.reduce(
```

Comment:
I expected this works to some extent with less computation. Seems like it's not. Will revert this.